staging/src/k8s.io/kubectl/pkg/validation/validation.go

```diff
@@ -27,19 +28,42 @@ import (
 	"k8s.io/kubectl/pkg/util/openapi"
 )
 
+// OpenAPIResourcesGetter represents a function to return
+// OpenAPI V2 resource specifications. Used for lazy-loading
+// these resource specifications.
+type OpenAPIResourcesGetter interface {
```

Comment:
Can we put this in a common package so smp can also use it?

Reply:
This interface now lives in the `k8s.io/kubectl/pkg/util/openapi` package (same as `Resources`).